ci/tests/Jenkinsfile.e2e-nightly

```diff
@@ -18,14 +18,19 @@ pipeline {
     disableConcurrentBuilds()
   }
 
-
   stages {
-    stage('Prep') {
-      steps { script {
-        if (params.PR_ID == null) {
-          error("PR_ID parameter not set!")
-        }
-      } }
+    stage('Fetch') {
+      steps { script { /* WARNING: This copies the latest available artifact or use specified APK_URL */
+          if (params.APK_URL != null) {
```

Comment:
The `params.APK_URL` is a string based on the definition in `parameters` so it's probably an empty string when not defined, not a `null`, so just do a `if (params.APK_URL)`.